### PR TITLE
[SVS] Add new recipes

### PR DESCRIPTION
**Pull request by A-Quarter-Mile**
Hi @ftshijt, here are new updates on Muskits-ESPnet:

- Add M4Singer recipe (multi singer)
- Add oniku recipe
- Add pjs recipe

### CODE REVIEW

File: egs2/m4singer/svs1/local/data_prep.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    "Soprano-1",
+    "Soprano-2",
+    "Soprano-3",
+    "Soprano-4",
+    "Tenor-1",
+    "Tenor-2",
+    "Tenor-3",
+    "Tenor-4",
+    "Tenor-5",
+    "Tenor-6",
```
Comment (ftshijt) [inline egs2/m4singer/svs1/local/data_prep.py:34]: Can you verify that the selected singer is the official test set?

Comment (A-Quarter-Mile) [reply egs2/m4singer/svs1/local/data_prep.py:34]: These are the singers in M4Singer. We can choose singers from this list according to different condition. It seems that there is no official split for test set.

Comment (ftshijt) [reply egs2/m4singer/svs1/local/data_prep.py:34]: sounds good

**Comment by ftshijt** [comment]
Could you please also add entries to the egs2/README.md?

**Comment by ftshijt** [comment]
Thanks for your contribution!

**Pull request merged**